Submodels/file_handling.py

```
@@ -173,10 +173,16 @@ def GetGridSize(self) -> list[float, float, float, float]:
                 raise TypeError("Expectred ducted_fan_design_params to be a dictionary.") from None
             if "Duct Leading Edge Coordinates" not in self.ducted_fan_design_params:
                 raise ValueError("Missing Duct Leading Edge Coordinates in design params") from None
+            if "Center Body Leading Edge Coordinates" not in self.ducted_fan_design_params:
+                raise ValueError("Missing Center Body Leading Edge Coordinates in design params") from None
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)